apps/retrospective/serializers/sprint.go

```diff
@@ -8,16 +8,35 @@ import (
 
 // Sprint is a serializer used in the Get sprint APIs
 type Sprint struct {
-	ID           uint
-	Title        string
-	SprintID     string
-	Status       int8
-	StartDate    time.Time
-	EndDate      time.Time
-	LastSyncedAt *time.Time
-	SyncStatus   int8
-	CreatedBy    userSerializer.User
-	CreatedByID  uint
+	ID              uint
+	Title           string
+	SprintID        string
+	Status          int8
+	StartDate       time.Time
+	EndDate         time.Time
+	LastSyncedAt    *time.Time
+	SyncStatus      int8
+	CreatedBy       userSerializer.User
+	CreatedByID     uint
+	RetrospectiveID uint
+	Summary SprintSummary
+}
+
+type SprintSummary struct {
+	MemberCount      int
+	TotalAllocation  float64
+	TotalExpectation float64
+	Holidays         float64
+	TotalVacations   float64
+	TargetSP         float64
+	TaskTypes        map[string]SprintTaskTypeSummary
```

Comment:
`TaskTypesSummary` or `TaskSummary` is more apt.
My pred is `TaskSummary`. Please take call

Reply:
Changed to `TaskSummary`